### DISCUSSION TITLE
Save Image to disk (or generate byte array?)

### DISCUSSION THREAD

**Discussion by kpfaulkner** [hajimehoshi/ebiten#1744] [Q&A]
Hi 

Just wondering if it's possible to save an ebiten image as a bytearray (or to a writer etc etc).
I find the image manipulation useful not only for games (ie to screen) but want to try some image processing with it, but for that I need to be able to save to disk. Any hints on that? I can't find a way to get the underlying data.

Thanks

Ken

**Comment by hajimehoshi** [comment]
As `*ebiten.Image` is just an [`image.Image`](https://pkg.go.dev/image#Image), you can encode an Ebiten image with the standard libraries and save it on your disk.
